tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -153,14 +157,89 @@ def test_it(
                         "authority": TEST_SETTINGS["h_authority"],
                         "enableShareLinks": False,
                         "grantToken": grant_token_service.generate_token.return_value,
-                        "groups": [context.course.groupid.return_value],
+                        "groups": Any(),
                     }
                 ]
             },
             "mode": "basic-lti-launch",
             "rpcServer": {"allowedOrigins": ["http://localhost:5000"]},
         }
 
+    @pytest.mark.usefixtures("grouping_plugin")
+    def test_configures_the_client_with_course_group(
```

Comment:
Now test the things that change based on different values for the grouping type.